dags/solutions_team/configs/tensorflow/solutionsteam_tf_release_supported_config.py

```diff
@@ -14,12 +14,51 @@
 
 """Utilities to construct configs for solutionsteam_tf_nightly_supported DAG."""
 
+from __future__ import annotations
+
 from xlml.apis import gcp_config, metric_config, task, test_config
 from dags import gcs_bucket, test_owner
 from dags.solutions_team.configs.tensorflow import common
 from airflow.models import Variable
 from dags.vm_resource import TpuVersion, Project, RuntimeVersion
-from typing import List
+
+
+MAJOR_VERSION = "2"
```

Comment:
I feel this part can be passed in from dag as params for `install_tf()`, `set_up_keras()`, and `set_up_tensorflow_models()`? So those methods could actually moved to `common.py`? 

```
MAJOR_VERSION = "2"
MINOR_VERSION = "16"
PATCH_VERSION = "1"
LIBTPU_VERSION = "1.10.0"
KERAS_VERSION = "2.16.0rc0"
MODELS_BRANCH = "r2.16.0"
```

Reply:
That's a good point, could add a line or two in the common versions of those functions rather than completely replicating them. I'll see what I can do.